e2e/panels-suite/panelEdit_base.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('Panel edit tests', () => {
         e2e.components.Tab.active().within((li: JQuery<HTMLLIElement>) => {
           expect(li.text()).equals('Transform data0'); // there's no transform so therefore Transform + 0
         });
-        e2e.components.Transforms.card('Merge').scrollIntoView().should('be.visible');
+        e2e.components.Transforms.card('Add field from calculation').scrollIntoView().should('be.visible');
```

Comment:
The `merge` transformation is no longer "visible" since there aren't 2 series available for `merge` to be "pickable."

Reply:
That's what I thought as well, but it is nevertheless not found the `be.visable` assertion. I think the `Add field from calculation` is a better de facto test, since it will always be available, but I can look into building out a better test for merge - or series of tests for the other transformations - if we think it's worth it. Thoughts?

Reply:
Looks like it was a naming issue after all. I updated the test to reflect the new title for merge: `Merge` -> `Merge series/tables`.